src/log.h

```diff
@@ -67,7 +78,9 @@ class Log {
                          const location&,
                          std::ostream&,
                          const std::string&);
+  std::string log_format_output(LogType, std::string&&);
   std::unordered_map<LogType, bool> enabled_map_;
+  bool is_colorize_;
```

Comment:
Be good to default it in case it's being run in unit tests.

```suggestion
  bool is_colorize_ = false;
```

Reply:
Done